src/actions/schedule-actions.js

```diff
@@ -7,30 +7,77 @@ export const UPDATE_FILTER      = 'UPDATE_FILTER';
 export const UPDATE_FILTERS     = 'UPDATE_FILTERS';
 export const CHANGE_VIEW        = 'CHANGE_VIEW';
 export const CHANGE_TIMEZONE    = 'CHANGE_TIMEZONE';
-
+/**
+ * This action is defined to just reinitialize the allScheduleReducer state
+ * (allSchedulesReducer.state.schedules) without trigerring the side effects of
+ * SYNC_DATA that reloads all the reducers to its initial state
+ * @type {string}
+ */
+export const RELOAD_SCHED_DATA = 'RELOAD_SCHED_DATA';
+/**
+ * this action is defined to reload the user schedules after the schedule data is
+ * initialized (allSchedulesReducer.state.schedules.length > 0 )
+ * @type {string}
+ */
+export const RELOAD_USER_PROFILE = 'RELOAD_USER_PROFILE';
 
 const fragmentParser = new FragmentParser();
 
 export const updateFilter = (key, filter, action = UPDATE_FILTER) => (dispatch) => {
     dispatch(createAction(action)({...filter, key}))
 };
 
-export const updateFiltersFromHash = (key, filters, view, actionCallback = UPDATE_FILTERS) => (dispatch) => {
-    const qsFilters = fragmentParser.getParams();
 
-    // clear hash
-    fragmentParser.clearParams();
+export const deepLinkToEvent = () => {
+    if (typeof window === "undefined") return null;
 
-    if (typeof window !== 'undefined') {
-        window.history.replaceState(null, null, ' ');
-    }
+    const eventHash = fragmentParser.getParam("event");
 
-    // escape if no hash
-    if (isEmpty(qsFilters)) return null;
+    if (eventHash) {
+        // when event=live we scroll to live line
+        const eventId = eventHash === "live" ? "live-line" : `event-${eventHash}`;
+        const eventEl = document.getElementById(eventId);
+
+        if (eventEl) {
+            // avoid scroll cache
+            window.history.scrollRestoration = "manual";
+
+            // remove event from fragment
+            fragmentParser.deleteParam("event");
+
+            // scroll to event
+            setTimeout(() => {
+                eventEl.scrollIntoView({ behavior: "smooth", block: "center" });
```

Comment:
Smooth scrolling still needs polyfilling in Safari.
Luckily was recently introduced in codebase [here](https://github.com/OpenStackweb/osf-website/pull/225/files#diff-ad9ea87f8488901c3102d09f1e42c10d60c96aecbce6cab0edccfc1b4faf5fccR9).

Reply:
ack